src/app/core/services/user/user.service.ts

```diff
@@ -12,6 +12,7 @@ const routes = {
   register: '/users/register',
```

Comment:
should move router config into another file

Reply:
I think that separate to another file is not necessary because the routes are not too much heavy. Instead of, the user routes inside the user.service.ts for easier to manage and update.